ReportPrinter.cs

```diff
@@ -4,10 +4,12 @@ public class ReportPrinter
     {
         public ReportCalculator ReportCalculator { get; set; }
         bool ReportIsNotSet { get; set; }
+        public CurrencyType currency { get; set; }
```

Comment:
I can't see the definition of the CurrencyType, I tried to search for it in the symbols but didn't find it